### ISSUE TITLE
Injected entries don't have task_name set

### ISSUE THREAD

**Issue by kolyagora** [Flexget/Flexget#1547]
### Expected behaviour:

A correct notification message should be sent on injecting an entry.

### Actual behaviour:

Notification title is set to `{{ task_name }}` instead of actual task name. Title is correct when the task runs by `execute` command.

### Steps to reproduce:
`flexget --debug inject FooBar`

#### Config:
```
tasks:
  TEST:
    mock:
      - {title: FooBar}
    accept_all: yes
    pushbullet:
      api_key: xxx
```
  
#### Log:
```
(Flexget) D:\devel\projects\flexget\Flexget>flexget --debug inject FooBar
2016-12-11 00:21 DEBUG    manager                       Figuring out config load paths
2016-12-11 00:21 DEBUG    manager                       Adding virtualenv path
2016-12-11 00:21 DEBUG    manager                       Found config: D:\devel\projects\flexget\Flexget\config.yml
2016-12-11 00:21 DEBUG    manager                       Config file D:\devel\projects\flexget\Flexget\config.yml selected
2016-12-11 00:21 DEBUG    manager                       sys.defaultencoding: ascii
2016-12-11 00:21 DEBUG    manager                       sys.getfilesystemencoding: mbcs
2016-12-11 00:21 DEBUG    manager                       os.path.supports_unicode_filenames: True
2016-12-11 00:21 DEBUG    plugin                        Trying to load plugins from: [u'D:\\devel\\projects\\flexget\\Flexget\\plugins', 'd:\\devel\\projects\\flexget\\flexget\\flexget\\plugins']
2016-12-11 00:21 DEBUG    safe                          Reading from cache file c:\users\nikolay\appdata\local\temp\safe-0.4.words.cache
2016-12-11 00:21 DEBUG    deluge                        Looking for deluge install in C:\Program Files (x86)\Deluge
2016-12-11 00:21 DEBUG    plugin                        (u'Plugin `%s` requires `%s` to load.', u'memusage', u'ext lib `guppy`')
2016-12-11 00:21 DEBUG    plugin                        Plugins took 2.51 seconds to load. 265 plugins in registry.
2016-12-11 00:21 DEBUG    manager                       Connecting to: sqlite:///D:\\devel\\projects\\flexget\\Flexget\\db-config.sqlite
2016-12-11 00:21 DEBUG    manager                       config_name: config
2016-12-11 00:21 DEBUG    manager                       config_base: D:\devel\projects\flexget\Flexget
2016-12-11 00:21 DEBUG    manager                       New config data loaded.
2016-12-11 00:21 DEBUG    schema                        entering flexget version 2.8.2.dev to db
2016-12-11 00:21 DEBUG    parsing                       setting default movie parser to internal. (options: {u'internal': <flexget.plugins.parsers.parser_internal.ParserInternal object at 0x04A24110>, u'guessit': <flexget.plugins.parsers.parser_guessit.ParserGuessit object at 0x050BAF30>})
2016-12-11 00:21 DEBUG    parsing                       setting default series parser to internal. (options: {u'internal': <flexget.plugins.parsers.parser_internal.ParserInternal object at 0x04A24110>, u'guessit': <flexget.plugins.parsers.parser_guessit.ParserGuessit object at 0x050BAF30>})
2016-12-11 00:21 DEBUG    cron_env                      Encoding mbcs stored
2016-12-11 00:21 DEBUG    util.simple_persistence                 setting key terminal_encoding value 'mbcs'
2016-12-11 00:21 INFO     ipc.rpyc                      server started on [127.0.0.1]:58294
2016-12-11 00:21 DEBUG    task_queue                    task queue shutdown requested
2016-12-11 00:21 DEBUG    task          TEST            executing TEST
2016-12-11 00:21 DEBUG    task          TEST            Disabling input phase
2016-12-11 00:21 DEBUG    status        TEST            Adding new task TEST
2016-12-11 00:21 INFO     task          TEST            Plugin mock is not executed because input phase is disabled (e.g. --test)
2016-12-11 00:21 VERBOSE  task          TEST            ACCEPTED: `FooBar` by accept_all plugin
2016-12-11 00:21 DEBUG    urlrewriter   TEST            Checking 1 entries
2016-12-11 00:21 VERBOSE  details       TEST            Summary - Accepted: 1 (Rejected: 0 Undecided: 0 Failed: 0)
2016-12-11 00:21 DEBUG    utils.template TEST            Error during rendering: (UndefinedError) 'task_name' is undefined
2016-12-11 00:21 DEBUG    notify        TEST            cannot render: `title: {{ task_name }}`, error: (UndefinedError) 'task_name' is undefined
2016-12-11 00:21 DEBUG    notify        TEST            trying to render default value for `title`
2016-12-11 00:21 DEBUG    utils.template TEST            Error during rendering: (UndefinedError) 'task_name' is undefined
2016-12-11 00:21 WARNING  notify        TEST            default dict failed to render: `title: {{ task_name }}`. Error: (UndefinedError) 'task_name' is undefined. Reverting to original value.
2016-12-11 00:21 DEBUG    notify        TEST            successfully rendered `url` to 
2016-12-11 00:21 DEBUG    notify        TEST            successfully rendered `message` to FooBar
2016-12-11 00:21 DEBUG    notify        TEST            successfully rendered `api_key` to xxx
2016-12-11 00:21 DEBUG    notify        TEST            Sending a notification to `pushbullet`
2016-12-11 00:21 DEBUG    utils.requests TEST            Fetching URL https://api.pushbullet.com/v2/pushes with args () and kwargs {'headers': {u'Content-Type': u'application/json', u'Accept': u'application/json', u'Authorization': 'Basic xxx=', u'User-Agent': u'Flexget'}, 'json': {u'body': u'FooBar', u'type': u'note', u'title': u'{{ task_name }}'}, 'data': None, u'timeout': 30}
2016-12-11 00:21 DEBUG    pushbullet    TEST            Pushbullet notification sent. Database operations remaining until next reset: 16216. Next reset at: 2016-12-11 00:50:33
2016-12-11 00:21 VERBOSE  notify        TEST            Successfully sent a notification to `pushbullet`
2016-12-11 00:21 DEBUG    seen          TEST            Learned 'FooBar' (field: title, local: 0)
2016-12-11 00:21 DEBUG    seen          TEST            Learned 'http://localhost/inject/Ev1gGPdxRFVbq5lcLS2AaYisnjBH34' (field: url, local: 0)
2016-12-11 00:21 DEBUG    util.simple_persistence TEST            Flushing simple persistence for task TEST to db.
2016-12-11 00:21 DEBUG    task_queue                    task queue shut down
2016-12-11 00:21 INFO     ipc.rpyc                      listener closed
2016-12-11 00:21 DEBUG    util.simple_persistence                 Flushing simple persistence for task None to db.
2016-12-11 00:21 DEBUG    manager                       Shutting down
2016-12-11 00:21 DEBUG    manager                       Removed D:\devel\projects\flexget\Flexget\.config-lock
```

### Additional information:

- Flexget Version: 2.8.2.dev
- Python Version: 2.7.10
- Installation method: Scripts/pip install -e .
- OS and version: Windows 8.1

**Comment by kolyagora** [comment]
Seems fixed after notification system changes in 2.9.0.

**Comment by kolyagora** [comment]
Seems fixed after notification system changes in 2.9.0.
